src/js/modules/rpc/errors.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+import {
+  DisableCoprocessorData,
+  EnableCoprocessor,
+  EnableCoprocessorMetadataItem,
+  EnableCoprocessorRequestData,
+} from "../domain/generatedRpc/generatedClasses";
+import { Handle } from "../domain/Handle";
+import { PolicyInjection } from "../public/Coprocessor";
+
+export enum EnableResponseCodes {
+  success,
+  internalError,
+  invalidIngestionPolicy,
+  scriptIdAlreadyExists,
+  scriptContainsInvalidTopic,
+  scriptContainsNoTopics,
+  scriptContainsSyntaxError,
+}
+
+export enum DisableResponseCode {
+  success,
+  internalError,
+  scriptDoesNotExist,
+}
+
+const maxSizeTopicName = 249;
+const validateTopic = (topic: string): boolean => {
```

Comment:
validateTopicName can be more descriptive

Reply:
or even `validateKafkaTopicName`

Reply:
This also needs a unit test

Reply:
Left a comment